app/src/main/java/com/asfoundation/wallet/subscriptions/list/SubscriptionListPresenter.kt

```diff
@@ -40,13 +41,18 @@ class SubscriptionListPresenter(private val view: SubscriptionListView,
   }
 
   private fun onSubscriptions(subscriptionModel: SubscriptionModel) {
+    val activeSubs = filterActive(subscriptionModel.allSubscriptions)
+    val expiredSubs = subscriptionModel.expiredSubscriptions
+    val bothEmpty = activeSubs.isEmpty() && expiredSubs.isEmpty()
     when {
       subscriptionModel.error == Error.NO_NETWORK -> if (!view.hasItems()) view.showNoNetworkError()
```

Comment:
I think we should we should double check on the `when` clauses and confirm that all the cases are covered.